Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
+This is how you would change the second line.
+This is the third line.
```

Comment:
Testing a reply.

---
In reply to: [59087918](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r59087918) [](ancestors = 59087918)